src/libraries/Microsoft.Extensions.Configuration.Json/src/JsonConfigurationProvider.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
-using System.Collections.Generic;
```

Comment:
Can you revert this in this change? Unnecessary changes should be avoided so we don't clutter source control history.